src/components/utils/__integrations__/onfidoApi/selfie.integration.js

```diff
@@ -155,7 +156,7 @@ describe('API uploadSnapshot endpoint', () => {
     expect.hasAssertions()
     const onErrorCallback = (error) => {
       try {
-        expect(error).toEqual(EXPECTED_EXPIRED_TOKEN_ERROR)
+        expect(error).toEqual(ORIGINAL_EXPECTED_EXPIRED_TOKEN_ERROR)
```

Comment:
Should this be `EXPECTED_EXPIRED_TOKEN_ERROR`?

Reply:
Oops. You're right!

Reply:
Not for `/snapshot` endpoint. That is still returning the original `expired_token` error response.

Reply:
Damn this is annoying 😅

Reply:
Agreed. 😅